WordPress/Classes/ViewRelated/Reader/Detail/ReaderDetailCoordinator.swift

```diff
@@ -188,8 +194,23 @@ class ReaderDetailCoordinator {
     /// Fetch Comments for the current post.
     ///
     func fetchComments(for post: ReaderPost) {
-        // TODO: fetch comments.
-        view?.updateComments()
+        commentService.syncHierarchicalComments(for: post,
+                                                   topLevelComments: commentsDisplayed,
+                                                   success: { [weak self] _, totalComments in
```

Comment:
Nothing to be fixed here, just wanted to point out yet another little Xcode 13 bug. The IDE recognizes `for` as a for-loop syntax instead of a method parameter, and it misaligns the other parameters on the next lines 😅 .

Reply:
> The IDE recognizes for as a for-loop syntax

Well of course it did. 😆 Thanks!